lib/plugins/aws/deployFunction/index.js

```diff
@@ -62,7 +66,8 @@ class AwsDeployFunction {
     return BbPromise.resolve();
   }
 
-  zipFunction() {
+  packageFunction() {
+    this.serverless.cli.log(`Packaging function: ${this.options.function}...`);
```

Comment:
@doapp-ryanp to be precise an ellipsis is a character that looks like three dots

… <- ellipsis
... <- 3 dots